docker-compose.yml

```diff
@@ -44,6 +45,10 @@ services:
       - mysql
       - wordpress
     command: tail -f /dev/null
+    environment:
+      WORDPRESS_DEBUG: 1
+      WORDPRESS_DB_USER: ${WP_DB_USER}
+      WORDPRESS_DB_PASSWORD: ${WP_DB_PASSWORD}
```

Comment:
The latest WP container image relies on environment variables. Ensure the CLI container is aware of the config used for the main WP/Apache container.